stores/evga.py

```diff
@@ -0,0 +1,169 @@
+            raise e
+
+        self.login(username, password)
+
+    def login(self, username, password):
+        """
+        We're just going to enter the user info and let the user handle the captcha
+        :param username:
+        :param password:
+        :return:
+        """
+        self.driver.execute_cdp_cmd(
+            "Network.setUserAgentOverride",
+            {
+                "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.53 Safari/537.36"
+            },
+        )
+        self.driver.execute_script(
+            "Object.defineProperty(navigator, 'webdriver', {get: () => undefined})"
+        )
+
+        self.driver.get(LOGIN_URL)
+        selenium_utils.wait_for_page(self.driver, "EVGA - Intelligent Innovation")
+
+        selenium_utils.field_send_keys(self.driver, "evga_login", username)
+        selenium_utils.field_send_keys(self.driver, "password", password)
+
+        log.info("Go do the captcha and log in")
+
+        selenium_utils.wait_for_page(
+            self.driver, "EVGA - Intelligent Innovation - Official Website", 300
+        )
+        log.info("Logged in!")
+
+    def buy(self, delay=5):
+        self.driver.get(
+            "https://www.evga.com/products/productlist.aspx?type=0&family=GeForce+30+Series+Family&chipset=RTX+3080"
+        )
+        # self.driver.get("https://www.evga.com/products/ProductList.aspx?type=0&family=GeForce+16+Series+Family&chipset=GTX+1660")
+
+        selenium_utils.wait_for_page(
+            self.driver,
+            "EVGA - Products - Graphics - GeForce 30 Series Family - RTX 3080",
+        )
+        # selenium_utils.wait_for_page(self.driver, "EVGA - Products - Graphics - GeForce 16 Series Family - GTX 1660")
+
+        #  Check for stock
+        log.info("On GPU Page")
+        atc_buttons = self.driver.find_elements_by_xpath(
+            '//input[@class="btnBigAddCart"]'
+        )
+        while not atc_buttons:
+            log.debug("Refreshing page for GPU")
+            self.driver.refresh()
+            atc_buttons = self.driver.find_elements_by_xpath(
+                '//input[@class="btnBigAddCart"]'
+            )
+            sleep(delay)
+
+        #  Add to cart
+        atc_buttons[0].click()
+
+        #  Go to checkout
+        selenium_utils.wait_for_page(self.driver, "EVGA - Checkout")
+        selenium_utils.button_click_using_xpath(
+            self.driver, '//*[@id="LFrame_CheckoutButton"]'
+        )
+
+        # Shipping Address screen
+        selenium_utils.wait_for_page(self.driver, "Shopping")
+
+        log.info("Skip that page.")
+        self.driver.get("https://secure.evga.com/Cart/Checkout_Payment.aspx")
+
+        selenium_utils.wait_for_page(self.driver, "EVGA - Checkout - Billing Options")
+
+        log.info("Ensure that we are paying with credit card")
+        WebDriverWait(self.driver, 10).until(
```

Comment:
I get an exception here

```
ElementClickInterceptedException
Message: element click intercepted: Element <input id="rdoCreditCard" type="radio" name="ctl00$LFrame$rdoPaymentType" value="rdoCreditCard"> is not clickable at point (1125, 1140). Other element would receive the click: <div class="ajax-bg">...</div>
  (Session info: chrome=85.0.4183.102)
```

From brief googling it might have something to do with AJAX being a cluster, but really I don't know. I'm on MacOS btw.

Reply:
I'd like to avoid using a hardcoded sleep, but I think I'll merge this and have an issue for it to fix later in the interest of getting this out fast